src/dbus/server/dbus_thread_object.cpp

```diff
@@ -830,7 +830,7 @@ otError DBusThreadObject::GetChannelMonitorAllChannelQualities(DBusMessageIter &
 exit:
     return error;
 #else  // OPENTHREAD_CONFIG_CHANNEL_MONITOR_ENABLE
-    (void)aIter;
+    OTBR_UNUSED_VARIABLE(aIter);
     return OT_ERROR_NOT_IMPLEMENTED;
```

Comment:
Not related to this PR, we may expect only one return statement? @bukepo @wgtdkp 
https://github.com/openthread/ot-br-posix/blob/master/STYLE_GUIDE.md
>Return Statements
There should be one return statement per free function or method at the end of the free function or method.